lib/matplotlib/text.py

```diff
@@ -239,10 +240,16 @@ def update(self, kwargs):
         """
         Update properties from a dictionary.
         """
-        bbox = kwargs.pop('bbox', None)
```

Comment:
I think "putting the object at the end of the dict" makes the intent clearer?  No strong opinion though, let me know if you really prefer your approach.